app/views/miq_policy_set/_profile_details.html.haml

```diff
@@ -1,113 +1,41 @@
-- if @edit
-  - url = url_for_only_path(:action => 'profile_field_changed', :id => (@profile.id || 'new'))
-  - observe_with_interval = {:interval => '.5', :url => url}.to_json
-- if @profile
-  #profile_info_div
-    = render :partial => "layouts/flash_msg"
-    - if @edit
-      -# Don't show basic info box with description only unless in edit mode
-      %h3
-        = _("Basic Information")
-      .form-horizontal
-        .form-group
-          %label.control-label.col-md-2
-            = _("Description")
-          - if @edit
-            .col-md-8
-              = text_field_tag("description", @edit[:new][:description],
-                :maxlength         => ViewHelper::MAX_DESC_LEN,
-                :class             => "form-control",
-                "data-miq_observe" => observe_with_interval)
-            = javascript_tag(javascript_focus('description'))
-          - else
-            .col-md-8
-              = h(@profile.description)
-      %hr
+- profile_policies = @record.miq_policies.sort_by { |p| [p.towhat, p.mode, p.description.downcase] } if @record
+#profile_info_div
+  = render :partial => "layouts/flash_msg"
+  -# Don't show basic info box with description only unless in edit mode
+  %h3
+    = _("Basic Information")
+  .form-horizontal
+    .form-group
+      %label.control-label.col-md-2
+        = _("Description")
+      .col-md-8
+        = h(@record.description)
+  %hr
 
-    - if @edit
-      %h3= _("Policy Selection")
-      .col-md-5
-        = _("Available Policies:")
-        %span#choices_chosen_div
-          = select_tag('choices_chosen[]',
-            options_for_select(@edit[:choices].sort),
-            :multiple => true,
-            :class    => "form-control",
-            :style    => "overflow-x: scroll;",
-            :size     => 8,
-            :id       => "choices_chosen")
+  %h3= _("Policies")
+  - if profile_policies.empty?
+    = render :partial => 'layouts/info_msg', :locals => {:message => _("No Policies are defined.")}
+  - else
+    %table.table.table-striped.table-bordered.table-hover
+      %tbody
+        - profile_policies.each do |p|
+          %tr{:class => "no-hover"}
+            -# uncomment following 2 lines when de-explorizing
+            -#%tr{:title => _("View this %{model} Policy") % {:model => ui_lookup(:model => p.towhat)},
+            -#:onclick => remote_function(:url => "/miq_policy/show/xx-#{p.mode}_xx-#{p.mode}-#{p.towhat}_p-#{p.id}")}
```

Comment:
Umm, aren't you de-explorerizing here?

Reply:
@skateman yes, but Polcy explorer is not de-explorized yet, so i am going to add all the links back in all these non-explorized versions once they are merged in one single PR.

Reply:
:+1: